server/controllers/notificationsController.js

```diff
@@ -0,0 +1,47 @@
+import '@babel/plugin-transform-regenerator';
+import '@babel/polyfill';
+import dotenv from 'dotenv';
+import notificationsService from '../services/notificationService';
+import pagination from '../helpers/paginationHelper';
+import Util from '../helpers/util';
+import { debugLogger }from '../helpers/loggerHandle';
+
+const util = new Util();
+dotenv.config();
+
+class NotificationsController {
+static async viewAllNotifications(req,res){
+  try{
+    let notifications;
+    if(req.userData.role === 'Manager'){
+       notifications = await notificationsService.findByProps(req.userData.id);
+  };
+  if(req.userData.role === 'Requester'){
+    notifications = await notificationsService.findByProp({user_id: req.userData.id});
+  };
```

Comment:
@bdushimi 
The endpoint allows only the `Requester` and `Manager` but in the notification controller, I had to check for the user's role, because to retrieve the manager's notifications, require a different query to that of the requester.